docs/requirements/apod/030-controlplane-bgp.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Documentation     Controlplane VMs need BGP peering with upstream devices.
+...
+...               Controlplane VM uses BGP for dynamic routing with upstream devices. 
+...               Usually it peers with 2 upstream devices for redundancy. 
+...               
+...               Controlplane VMs should use same next-hop IP addresses within management network for 
+...               BGP peering with 2 upstream devices independant of pod.
+...               
+...               Controlplane VMs use traceroute with ttl=1 for upstream device IP address discovery 
+...               and use discovered IP addresses for their BGP next-hop configuration.
+...               
+...               BGP peering is used for prefix announcement from Controlplane VMs to upstream.
+...               
+...               Controlplane VMs advertise Kubernetes internal subnets which must not be routed outside 
+...               of the fabric. Communication between controlplane VMs and Kubernetes internal subnets 
+...               must be possible.
+...               
+...               Controlplane VMs advertise Kubernetes external subnets and /32 IP addresses which must
```

Comment:
Understood, is there a possibility to advertise pod-cidrs on a seperate peering than k8s services? This would probably be on another VLAN. In that VLAN and on that peering IP the default route should also be configured/received.